### DISCUSSION TITLE
[Question] is there any inbuilt way in flet to do device fingerprinting so that can detect even they don't login

### DISCUSSION THREAD

**Discussion by plugeit** [flet-dev/flet#1875] [Q&A]
### Question

I want to track users like using cookies etc. so that I can track their free trial usage which I am planning to give without requiring them to login.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
https://api.flutter.dev/flutter/dart-io/Cookie-class.html ?

**Comment by mouadessalim** [comment]
i've already created an issue for this (#1384) several months ago but without receiving any response.
